lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/evaluator/VortexWorker.java

```diff
@@ -97,30 +99,26 @@ public void run() {
           }
 
           // Command Executor: Deserialize the command
-          final VortexRequest vortexRequest = vortexAvroUtils.toVortexRequest(message);
+          final MasterToWorker masterToWorker = (MasterToWorker)kryoUtils.deserialize(message);
 
-          switch (vortexRequest.getType()) {
+          switch (masterToWorker.getType()) {
             case AggregateTasklets:
-              final TaskletAggregationRequest taskletAggregationRequest = (TaskletAggregationRequest) vortexRequest;
-              aggregates.put(taskletAggregationRequest.getAggregateFunctionId(),
-                  new AggregateContainer(heartBeatTriggerManager, vortexAvroUtils, workerReports,
-                      taskletAggregationRequest));
-
-              // VortexFunctions need to be put into the repository such that VortexAvroUtils will know how to
```

Comment:
Because we no longer use `VortexAvroUtils`, we don't need to store `VortexFunctions` in `AggregateFunctionRepository`. To make things less confusing, I also removed the related code in `AggregateFunctionRepository`.